pharmd-app/src/components/Nav/AppBar.js

```diff
@@ -35,16 +17,13 @@ const Toolbar = styled(MuiToolbar)`
   ${tw`absolute bottom-0`}
 `;
 
-//-------------------------- COMPONENT --------------------------
-
 const AppBar = ({ title, location, children, props }) => (
   <AppBarStyled color="transparent" position="relative" {...props}>
-    <Toolbar disableGutters={true}>
+    <Toolbar disableGutters>
       <div>
         <RouterBreadcrumb locTest={location} />
         <Title>{title}</Title>
       </div>
-      {/* Incase if any toolbar action buttons are added to the component */}
```

Comment:
I think this comment would be a useful description. I am a little confused as to what it is referring to though.